web/cypress/e2e/morecast-2-page.cy.ts

```diff
@@ -3,18 +3,19 @@ import { MORE_CAST_2_ROUTE } from '../../src/utils/constants'
 describe('More Cast 2 Page', () => {
   beforeEach(() => {
     cy.intercept('GET', 'api/fba/fire-centers', { fixture: 'fba/fire-centers.json' }).as('fireCenters')
+    cy.intercept('GET', 'api/stations/groups').as('stationGroups')
 
     cy.visit(MORE_CAST_2_ROUTE)
   })
 
   it('Renders the initial page', () => {
-    cy.getByTestId('fire-center-dropdown').should('be.visible')
+    cy.getByTestId('station-group-dropdown').should('be.visible')
     cy.getByTestId('morecast2-data-grid').should('be.visible')
     cy.getByTestId('morecast2-station-panel').should('be.visible')
     cy.getByTestId('weather-model-dropdown').should('be.visible')
   })
 
-  it('Writes typed fire center to local storage', () => {
+  xit('Writes typed fire center to local storage', () => {
```

Comment:
Should this test be removed or re-enabled?

Reply:
I think this one can be removed, and the other I'll add a comment to re-enable once we have the tabbed view in.

Reply:
I removed both since we're not storing fire centres anymore, let's see what users think about persisting/not persisting groups.